README.md

```diff
@@ -0,0 +1,51 @@
+# SES5
+
+> Secure EcmaScript 5
+
+*:warning: This is not an official, working packaging of Google Caja’s SES.
+This is a project attempting to bring SES to Node.js.  There remain a few
+surmountable obstacles before we achieve proper containment.*
+
+SES is a tool that allows mutually suspicious programs to share a single
+EcmaScript 5 compliant JavaScript context without interfering with each
+other.  It does this by freezing everything that is accessible in global
+scope, removing interfaces that would allow programs to interfe with
+each-other, and providing the ability to evaluate arbitrary code in
+isolation.
+
+SES is a part of the Google Caja project.  For JavaScript contexts that
+do not support EcmaScript 5, Caja depends on compiling JavaScript to a
+JavaScript subset with static verification and run-time assertions to
+maintain isolation.  With EcmaScript 5, it is possible to run isolated
+code without a compilation step or run-time checks.
+
+Usage:
+
+```js
+var ses = require('ses5');
```

Comment:
In demo.js you require 'ses' rather than 'ses5'. Don't you want to do that here too?